maxmind.json

```diff
@@ -5,7 +5,7 @@
     "type": "information",
     "publisher": "Splunk",
     "main_module": "maxmind_connector.py",
-    "app_version": "2.1.11",
+    "app_version": "2.2.0",
```

Comment:
Seeing that this is a bug fix, I believe we need to bump the minor version here.